graylog2-web-interface/src/views/components/aggregationwizard/ElementConfigurationContainer.tsx

```diff
@@ -62,17 +68,17 @@ type Props = {
 
 const ElementConfigurationContainer = forwardRef<HTMLDivElement, Props>(({ children, onRemove, testIdPrefix, dragHandleProps, className, draggableProps, elementTitle }: Props, ref) => (
   <Container className={className} ref={ref} {...(draggableProps ?? {})}>
-    <ElementConfiguration>
-      {children}
-    </ElementConfiguration>
     <ElementActions>
       {dragHandleProps && (
-        <DragHandle {...dragHandleProps} data-testid={`${testIdPrefix}-drag-handle`}>
-          <Icon name="drag_indicator" />
-        </DragHandle>
+      <DragHandle {...dragHandleProps} data-testid={`${testIdPrefix}-drag-handle`}>
+        <Icon size="sm" name="drag_indicator" />
+      </DragHandle>
       )}
-      {onRemove && <IconButton onClick={onRemove} name="delete" title={`Remove ${elementTitle}`} />}
+      {onRemove && <StyledIconButton size="sm" onClick={onRemove} name="close" title={`Remove ${elementTitle}`} />}
```

Comment:
I moved this icon to the top because I needed some space for a unit button. Now it's on the top right corner and on my opinion closing cross looks better there. Also, this icon looks different from the one we have for deleting the field in the group section. But if you thing we have to stick with delete I can roll it back.

Reply:
I see the point with the unique icon. I think it can be confusing though, when the trashcan icon means removing the field, but the cross is for removing the pivot (the cross can be mistaken for "closing" it).

Another issue is that we are now creating an otherwise empty row where the cross is, while leaving vertical space below it:

![image](https://github.com/user-attachments/assets/483c0145-6b2d-4970-89fd-eea2c66e6f2a)

Reply:
Ok. I can return the trash icon and change the styling a bit (add absolute and right: 0 )
<img width="431" alt="Screenshot 2024-08-12 at 16 13 39" src="https://github.com/user-attachments/assets/c002de15-cb1b-4f6c-9438-090786ba58b5">